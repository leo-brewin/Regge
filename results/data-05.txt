# defect
 8.068244055247735660E-03

# i  j   d(defect)/d(lsq(i,j))
  1  2 -5.499656050703699160E+00
  1  3 -5.503329986666803500E+00
  2  3 -5.656145402707055650E+00
  4  1  6.483139868013298170E+00
  4  2  6.776259359197646370E+00
  4  3  6.782966751100503370E+00
  5  1  8.976819602205482180E-01
  5  2  7.901887921505956530E-01
  5  3  7.879955228177107900E-01
  4  5 -5.281801112287956680E+00
  6  1  8.273660350887676110E-01
  6  2  8.232944889993807570E-01
  6  3  8.233031731202128410E-01
  5  6 -5.282984944881967760E+00
  7  1  7.467652951745835980E-01
  7  2  8.611103881963957670E-01
  7  3  8.635794172262314510E-01
  6  7 -5.289528917087185570E+00
  8  1  7.927825222981570740E-01
  8  2  8.392547459207928820E-01
  8  3  8.402662144321333890E-01
  7  8 -5.290982440282185660E+00
  9  1  6.601739858237415140E+00
  9  2  6.718159280134063510E+00
  9  3  6.721189957632563190E+00
  8  9 -5.284797216793682070E+00
  9  4 -9.708829591932238230E+00
