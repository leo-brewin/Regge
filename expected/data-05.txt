# defect
 8.068244055247734360E-03

# i  j   d(defect)/d(lsq(i,j))
  1  2 -5.499656050703699170E+00
  1  3 -5.503329986666803500E+00
  2  3 -5.656145402707055700E+00
  4  1  6.483139868013298160E+00
  4  2  6.776259359197646410E+00
  4  3  6.782966751100503410E+00
  5  1  8.976819602205482150E-01
  5  2  7.901887921505956610E-01
  5  3  7.879955228177107970E-01
  4  5 -5.281801112287956680E+00
  6  1  8.273660350887675910E-01
  6  2  8.232944889993807710E-01
  6  3  8.233031731202128540E-01
  5  6 -5.282984944881967760E+00
  7  1  7.467652951745836120E-01
  7  2  8.611103881963957650E-01
  7  3  8.635794172262314460E-01
  6  7 -5.289528917087185580E+00
  8  1  7.927825222981570860E-01
  8  2  8.392547459207928680E-01
  8  3  8.402662144321333840E-01
  7  8 -5.290982440282185660E+00
  9  1  6.601739858237415110E+00
  9  2  6.718159280134063570E+00
  9  3  6.721189957632563250E+00
  8  9 -5.284797216793682070E+00
  9  4 -9.708829591932238270E+00
